File: CSharp/FileControl.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.IO;

namespace CSharp
{
    class FileControl : IExecutable
    {
        public override void Question01()
        {
            FileStream fs = new FileStream("output.txt", FileMode.OpenOrCreate, FileAccess.ReadWrite);

            string output = "This is the string to write.\n";
            byte[] outputBytes = Encoding.Unicode.GetBytes(output);

            fs.Write(outputBytes);

            fs.Close();

            Console.WriteLine("Completed.");

        }

        public override void Question02()
        {
            throw new NotImplementedException();
        }

        public override void Question03()
        {
            throw new NotImplementedException();
        }

        public override void Question04()
        {
            throw new NotImplementedException();
        }
    }
}

File: Kotlin/src/FileControl.kt
import java.io.FileOutputStream
import java.io.IOException

class FileControl : IExecutable() {
    override fun question_01() {
        try {
            val fs = FileOutputStream("output.txt")
            val outputString = "This is the string to write.\n"
            fs.write(outputString.toByteArray())
            fs.close()
            println("Completed.")
        } catch (ex: IOException) {
            println(ex.message)
        }
    }

    override fun question_02() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun question_03() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun question_04() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}